types/filecontracts.go

```diff
@@ -108,15 +108,23 @@ func (fcid FileContractID) StorageProofOutputID(proofStatus ProofStatus, i uint6
 	))
 }
 
+// PostTax returns the amount of currency remaining in a file contract payout
+// after tax.
+func PostTax(height BlockHeight, payout Currency) Currency {
+	// COMPATv0.4.0 - until the first 20,000 blocks have been archived, they
+	// will need to be handled in a special way.
+	if (height < 21e3 && build.Release == "standard") || (height < 10 && build.Release == "testing") {
+		return payout.Sub(payout.MulFloat(0.039).RoundDown(SiafundCount))
+	}
+	return payout.Sub(payout.MulTax().RoundDown(SiafundCount))
+}
```

Comment:
I thought about making tax a method of the currency, but really it's about file contracts and not about the currency type. If the currency type were a separate package, the Tax method would not follow it. So I didn't think it should be of the currency type.

PostTax is used as a function almost as much as Tax is, perhaps even more than Tax is. I'm open to a different name but think it's valuable as a function.